articles/best-practices-for-rich-text.mdx

```diff
@@ -27,10 +27,35 @@ To prevent cross-site scripting attacks and other unwanted interactions between
 
 <comp.DocsImage path="/documentation/static/boxed-xss-elements.png" alt="script and style elements replaced with boxes" type="screenshot-snippet"/>
 
-Some elements with simple text content, like `kbd` and `main`, will be included in this process and will become uneditable. These elements typically also fall into the category of HTML that cannot be reconstructed, and should be avoided in rich text content anyway.
+Some elements with simple text content, like `kbd` and `main`, will be included in this process and will become uneditable. These elements always also fall into the category of HTML that cannot be reconstructed, and should be avoided in rich text content anyway.
+
+The full list of elements that will be replaced with boxes is: `base`, `blink`, `button`, `canvas`, `data`, `datalist`, `dfn`, `dialog`, `fieldset`, `form`, `hgroup`, `iframe`, `input`, `kbd`, `label`, `legend`, `link`, `main`, `map`, `marquee`, `math`, `menu`, `menuitem`, `meta`, `meter`, `noscript`, `object`, `optgroup`, `option`, `output`, `picture`, `portal`, `progress`, `q`, `rb`, `rp`, `rt`, `ruby`, `samp`, `script`, `select`, `slot`, `source`, `style`, `svg`, `template`, `textarea`, `time`, `title`, `track`, `var`, and `wbr`.
+
+Any content that is not boxed this way will also be processed with an XSS stripping library, which will remove any remaining potential attack vectors. This will remove most attributes from HTML elements. The only exceptions are the `class` attribute, and other basic attributes required for the functionality of different elements (e.g. `src` and `alt` for `img`). Once again, these attributes also fall into the category of HTML that cannot be reconstructed, and should be avoided in rich text content anyway.
```

Comment:
```suggestion
Any content that is not boxed this way will also be processed with a sanitization library, to remove potential vulnerabilities or clashes with the CloudCannon interface. This will remove most attributes from HTML elements. The only exceptions are the `class` attribute, and other basic attributes required for the functionality of different elements (e.g. `src` and `alt` for `img`). Once again, these attributes also fall into the category of HTML that cannot be reconstructed, and should be avoided in rich text content anyway.
```